### PR TITLE
Kumarabd/feature/helm charts

### LINKED ISSUE #2837: [Meshery] Helm chart access to be updated
#### Description
Currently meshery helm chart does not have the permissions to do operations on MeshSync custom resources.

#### Expected Behavior
The helm chart needs to be modified, to grant access to these custom resources. The access levels can be picked up from here - https://github.com/layer5io/meshery-operator/blob/master/config/manifests/default.yaml#L199-L320.

#### Screenshots/Logs
```
time="2021-05-05T06:29:13Z" level=error msg="Meshery kubernetes client not initialized.brokers.meshery.layer5.io \"meshery-broker\" is forbidden: User \"system:serviceaccount:meshery:meshery-service-account\" cannot get resource \"brokers\" in API group \"meshery.layer5.io\" in the namespace \"meshery\"" app=meshery code=test_code probable-cause=None severity=1 short-description=None suggested-remediation=None
```

#### Environment:
- Meshery Version: any
- Kubernetes Version: any
- Host OS: any
- Browser: any

### PR DESCRIPTION
**Pull request by kumarabd**
**Description**

This PR fixes #2837

@leecalcote Ive moved the charts back inside to have it as a dependency chart (This is recommended if the other charts depend on a parent chart to exist). 

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/kubernetes/helm/meshery/Chart.yaml [hunk 1]
```diff
@@ -52,10 +52,10 @@ dependencies:
   version: 2.0.1
   condition: meshery-kuma.enabled
 - name: meshery-osm
-  version: 1.0.1
+  version: 2.0.1
   condition: meshery-osm.enabled
 - name: meshery-nsm
-  version: 1.0.1
+  version: 2.0.1
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/Chart.yaml:58]: Can we using the current release tag?

**Comment by leecalcote** [comment]
We're in need of a Traefik Mesh chart. We can leave Octarine aside, but should probably include Citrix, too.

**Comment by leecalcote** [comment]
We'll want to include a chart for Meshery Operator, too.

**Comment by leecalcote** [comment]
Need to add instructions for the user to be able to switch channels between stable and edge.

**Comment by leecalcote** [comment]
meshery.io repo needs cleaned up with charts being removed. Verify that all looks good on Artifact Hub.

**Comment by leecalcote** [comment]
I _think_ we are all set here save for the final item of incorporating the (`ref/tag`) release tag.

**Comment by kumarabd** [comment]
@leecalcote All of the items are covered. Good to release.

File: install/kubernetes/helm/meshery/Chart.yaml [hunk 2]
```diff
@@ -11,6 +11,9 @@ keywords:
   - meshery-kuma
   - meshery-osm
   - meshery-nsm
+  - meshery-traefik-mesh
+  - meshery-cpx
+  - meshery-adapter
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/Chart.yaml:16]:
```suggestion
  - meshery-operator
```

**Review by leecalcote — COMMENTED** [review]
Is the build version still hardcoded or will it be changed to reflect the release tag when published?

**Review by leecalcote — APPROVED** [review]

**Pull request merged**